docs/ios.md

```diff
@@ -70,64 +78,115 @@ account.activate(completion: { txHash, error in
 })
 ```
 
-#### KIN
+## Account Information
 
-To retrieve the account's current balance:
+#### Public Address
 
 ```swift
-func balance(completion: @escaping BalanceCompletion)
+let publicAddress = account.publicAddress
 ```
 
-To obtain a watcher object which will emit an event whenever the account's balance changes.  See the Sample App for an example.
+The account's address on the network.  This is the identifier used to specify the destination for a payment, or to request account creation from a service.
+
+#### Account Status
 
 ```swift
-func watchBalance(_ balance: Decimal?) throws -> BalanceWatch
+account.status(completion: { status, error in
+    if let status == status {
+        switch status {
+        case notCreated: break
+        case notActivated: break
+        case activated: break
+        }
+    }
+})
 ```
 
-To send KIN to another user:
+#### Retrieving KIN Balance
 
 ```swift
-func sendTransaction(to recipient: String,
-                         kin: Decimal,
-                         memo: String?,
-                         completion: @escaping TransactionCompletion)
+account.balance(completion: { balance, error in
+    if let balance == balance {
+      ...
+    }
+})
 ```
 
-The `memo` field can contain a string up to 28 characters in length.  A typical usage is to include an order# that a service can use to verify payment.
+## Transactions
+
+To send KIN to another account, the account's public address is required.
+
+```swift
+account.sendTransaction(to "GDIRGGTBE3H4CUIHNIFZGUECGFQ5MBGIZTPWGUHPIEVOOHFHSCAGMEHO",
+                        kin: 123,
+                        memo: "payment for donuts",
+                        completion: { txHash, errors
+                          if let txHash = txHash {
+                            // payment succeeded
+                          }
+                        })
+```
 
----
+The `memo` field can contain a utf-8 string up to 28 bytes in length.  A typical usage is to include an order# that a service can use to verify payment.
```

Comment:
can you add this to kin-core-ios docs also? (code documentation)

Reply:
This is done.